src/common/components/vanilla-modules/button/index.js

```diff
@@ -22,7 +22,7 @@ function createButtonComponent(Component) {
     return (
       <Component {...rest} className={ className }>
         { isSpinner &&
-          <span className={ customStyle.spinner }><Spinner light/></span>
+          <IconSpinner color='inherit-color'/>
```

Comment:
This causes button to change the size when spinner appears.

Spinner in button needs to be placed in a wrapper that is positioned in the center of the button, and text needs to be transparent (not moved out with text indent).

```js
<span className={ customStyle.spinner }><IconSpinner color='inherit-color' /></span>
```